utopia-remix/server.js

```diff
@@ -191,3 +198,12 @@ app.listen(environment.PORT, listenCallback(environment.PORT))
 if (process.env.NODE_ENV === 'development') {
   app.listen(environment.PORT + 1, listenCallback(environment.PORT + 1))
 }
+
+// To make sure everythin keeps working and the server doesn't crash, if
+// there are any uncaught errors, log them out gracefully
+process.on('uncaughtException', (err) => {
```

Comment:
Probably want to put this at the top of this file, so they're setup before anything comes online and starts doing things.

Reply:
[`ccb086e` (#5052)](https://github.com/concrete-utopia/utopia/pull/5052/commits/ccb086efb0198a0347b5f6f72c145481703cd27b)